src/Components/context/UserContext.js

```diff
@@ -0,0 +1,15 @@
+import { createContext, useState } from 'react';
```

Comment:
Context is not a component, so I would bring your context folder out of your Components folder.